src/main/java/edu/asu/plp/tool/backend/isa/events/RegWatchResponseEvent.java

```diff
@@ -2,32 +2,74 @@
 
 import javafx.beans.property.LongProperty;
 
+/**
+ * @brief Event to denote Register Watch Result. It returns a reference of the register object.
+ * 	      It offers communication between the front-end and the back-end
+ */
 public class RegWatchResponseEvent {
 
+	/**
+	 * @brief Boolean Variable to store the status of the request result(True or False)
+	 */
 	private boolean success;
+
+	/**
+	 * @brief String variable to hold the registerID
+	 */
 	private String registerID;
+
+	/**
+	 * @brief String variable to hold the registerName
+	 */
 	private String registerName;
+
+	/**
+	 * @brief LongProperty class defines a Property wrapping a long value.
+	 */
 	private LongProperty regObject;
 
+	/**
+	 * @brief This is a constructor for which accepts Success, ErrorMessage, ProjectName and ASMImage parameters
```

Comment:
DO NOT COPY others work, show your understanding on the event-driven model.
<pre>This is a constructor for which accepts Success, <b>ErrorMessage</b>, <b>ProjectName</b> and <b>ASMImage</b> parameters.</pre>
Obviously, we don't have above parameters here.

Reply:
Was replicating Java Doc syntax. Mistakenly forgot to remove the comment. Will rectify and issue a new pull request.